one_fm/api/v1/authentication.py

```diff
@@ -285,12 +285,18 @@ def send_token_via_email(user, token, otp_secret, otp_issuer, subject=None, mess
 def validate_employee_id(employee_id=None):
 	if employee_id is None:
 		return response("Employee ID cannot be None", 401, None, "Employee ID is required !")
-	doc = frappe.db.get_value("Employee",{ "employee_id": employee_id}, "employee_name")
-	if doc == None:
+	doc = frappe.get_doc("Employee",{ "employee_id": employee_id})
+	if not doc:
 		return response("Employee Not Found", 404, None, "Employee ID of an active Employee is required")
-	return response("Success", 200, doc)
-
-
+	registration_status = frappe.db.get_value("User", doc.user_id, "last_password_reset_date")
```

Comment:
@NoBoneZ Add inline comments in the API arguments please

Reply:
Done